sbnobj/Common/Reco/SimpleFlashMatchVars.h

```diff
@@ -0,0 +1,45 @@
+#ifndef sbncode_Simpleflashmatchvars_H
+#define sbncode_Simpleflashmatchvars_H
+
+#include "TVector3.h"
+
+namespace sbn
+{
+  class SimpleFlashMatch
+  {
+  public:
+  SimpleFlashMatch(
+    bool present = false, double time = -1,
+    double charge_q = -1, double light_pe = -1,
+    double score = -1, double scr_y = -1, double scr_z = -1,
+    double scr_rr = -1, double scr_ratio = -1,
+    TVector3 chargeXYZ = TVector3(-999, -999, -999),
+    TVector3 lightXYZ = TVector3(-999,-999,-999)):
+    mPresent(present),
+    mTime(time),
+    mChargeQ(charge_q),
+    mLightPE(light_pe),
+    mScore(score),
+    mScr_y(scr_y),
+    mScr_z(scr_z),
+    mScr_rr(scr_rr),
+    mScr_ratio(scr_ratio),
+    mChargeXYZ(chargeXYZ),
+    mLightXYZ(lightXYZ)
+    {}
+
+    bool mPresent;
```

Comment:
The convention elsewhere in larsoft is `f` for field, rather than `m` for member. Unless this is matching the style of some related existing code?

Reply:
This was the convention in place in the old object when the edits were made although we're happy to change to a different convention. Personally I'm used to `f` in this instance as you mention as well.

Reply:
Looking at other `sbnobj` classes I see that it's fairly common to not have `m` or `f`. So I decided to remove clutter.